### DISCUSSION TITLE
why mount cache dir inside?

### DISCUSSION THREAD

**Discussion by avoidik** [abiosoft/colima#1146] [Q&A]
hello,

why we mount a cache dir inside of a colima vm? I'd like to understand the use-case.

https://github.com/abiosoft/colima/blob/a58166e288acc54cc98d890fad49a643203b04e8/environment/vm/lima/yaml.go#L323

```
$ mount | grep colima
mount0 on /Users/user.name/Library/Caches/colima type virtiofs (ro,relatime)
```

it seems like a bug to me, because the cache dir in this case is only used to check overlapping directories, and eventually should be removed from the `l.Mounts` list

regards

**Comment by abiosoft** [comment]
It is used as a means of sharing files between the host (macOS) and the guest (virtual machine). 
Colima uses it internally and in multiple places and it is abstracted from the end-users.

**Reply by avoidik** [reply]
It's ready only. Is it a one way channel?
